.gitignore

```diff
@@ -78,3 +78,4 @@ stryker.log
 # yalc
 .yalc/
 yalc.lock
+startnode.sh
```

Comment:
@ronaldmansveld - are we not going to have an `ignore` directory instead?

Reply:
Good shout, I'll do some cleanup around this